lib/tools/adb-emu-commands.js

```diff
@@ -2,7 +2,12 @@ import log from '../logger.js';
 import _ from 'lodash';
 
 const PHONE_NUMBER_PATTERN = /^[\+]?[(]?[0-9]*[)]?[-\s\.]?[0-9]*[-\s\.]?[0-9]{2,}$/im;
+
 let emuMethods = {};
+emuMethods.POWER_AC_STATES = {
+  'POWER_AC_ON': 'on',
```

Comment:
No need for Python-ish quotes around keys, unless they are invalid JS variable names.